Dockerfile

```diff
@@ -1,13 +1,21 @@
 FROM php:7.1-apache
+MAINTAINER "demian.katz@villanova.edu"
```

Comment:
@demiankatz I'm happy to put my contact info here if you don't want to be on the hook for the Docker stuff quite yet. :)